Frontend/src/components/Curso/FooterCursos.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+class Footer extends Component {
+  render() {
+    const { price, rating } = this.props;
+    let estrellas = "";
+
+    for (var i = 0; i < Math.round(rating); i++) {
```

Comment:
Extract the start creation functionality into a function outside of the component

Reply:
Ready